### PR TITLE
Integrate RL skills for real-world experiments

### PR DESCRIPTION
**Pull request by yvsriram**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

# Checklist:

- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] I ran `tests/hw_manual_test.py` on hardware and observed that robot behavior looks normal.

### CODE REVIEW

**Review by cpaxton — COMMENTED** [review]
some comments - lots of stuff that i couldnt quite follow

File: src/home_robot/home_robot/agent/ovmm_agent/complete_obs_space.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import gym.spaces as spaces
+import numpy as np
+
+
+def get_complete_obs_space(skill_config, baseline_config):
```
[the remark this answers is not in the document]
Comment (yvsriram) [reply src/home_robot/home_robot/agent/ovmm_agent/complete_obs_space.py:5]: This is the global observation space that is used by the policies. We plan to load this from configs.

File: src/home_robot/home_robot/agent/ovmm_agent/ovmm_agent.py [hunk 2]
```diff
@@ -112,9 +120,11 @@ def reset_vectorized_for_env(self, e: int, episode):
             self.gaze_agent.reset_vectorized_for_env(e)
         if self.nav_to_obj_agent is not None:
             self.nav_to_obj_agent.reset_vectorized_for_env(e)
+        if self.place_agent is not None:
+            self.place_agent.reset_vectorized_for_env(e)
```
[the remark this answers is not in the document]
Comment (yvsriram) [reply src/home_robot/home_robot/agent/ovmm_agent/ovmm_agent.py:124]: We should use a uniform function prototype for all policies. For now, I will just rename them to be the same.

File: src/home_robot/home_robot/agent/ovmm_agent/ppo_agent.py [hunk 3]
```diff
@@ -246,17 +223,38 @@ def reset_vectorized_for_env(self, e: int):
...
-        # raise NotImplementedError
-        # For Gaze check if the center pixel corresponds to the object of interest
-        if not self.continuous_actions:
-            return actions == DiscreteNavigationAction.STOP
-        h, w = observations.semantic.shape
-        return (
-            observations.semantic[h // 2, w // 2]
-            == observations.task_observations["object_goal"]
-        )
+    def does_want_terminate(self, observations, action) -> bool:
```
[the remark this answers is not in the document]
Comment (yvsriram) [reply src/home_robot/home_robot/agent/ovmm_agent/ppo_agent.py:226]: Will do

**Comment by yvsriram** [comment]
> some comments - lots of stuff that i couldnt quite follow
Yes, sorry didn't make this ready. Thanks for reviewing. These are the items from the review:. 

1. loading global observation space from configs
2. adding comments describing the observation space
3. rename place agent and place policy to be the same
4. add comments for terminate condition 
5. rename heuristic_place_debug to heuristic_place
6. confirm clip dependency

**Comment by yvsriram** [comment]
Will handle this in a fresh PR now

**Pull request closed without merging**